src/pages/GovPages/Proposals/index.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useContext } from 'react'
+import styled, { ThemeContext } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import { Flex, Text } from 'rebass'
+
+import { TYPE } from '../../../theme'
+import { PageWrapper } from '../../Pool/styleds'
+import { useRouter } from '../../../hooks/useRouter'
+
+import { LightCard } from '../../../components/Card'
+import { RowBetween } from '../../../components/Row'
+import { AutoColumn } from '../../../components/Column'
+import CurrencyLogo from '../../../components/CurrencyLogo'
+import { ButtonPrimary, ButtonSecondary } from '../../../components/Button'
+import TabBar from '../../../components/TabBar'
+import Proposals from './Proposals'
+
+// import { Redirect } from 'react-router-dom'
+
+const ContentCard = styled(LightCard)`
+  background: linear-gradient(113.18deg, rgba(255, 255, 255, 0.35) -0.1%, rgba(0, 0, 0, 0) 98.9%),
+    ${({ theme }) => theme.dark1};
+  background-blend-mode: overlay, normal;
+  border-radius: 8px;
+  padding: 25px 20px;
+  position: relative;
+  border: none;
+
+  ::before {
```

Comment:
Same question above, what's the role of this? I've used ::before pseudoelements in the past to implement fade in/out of gradients on hover, but I don't see any `opacity` set here.

Reply:
This is for the border of proposal card.